src/components/Layout/OurTeam.tsx

```diff
@@ -1,14 +1,112 @@
-import React, { useState, useEffect } from "react";
-import { Center, Container, Flex, Grid, GridItem, Text } from "@chakra-ui/react";
+import React from "react";
+import {
+  Box,
+  Container,
+  Flex,
+  Heading,
+  SimpleGrid,
+  Text
+} from "@chakra-ui/react";
 
+const teamMembers = [
+  {
+    name: "Anna Pestova",
+    role: "Front-end",
+    imageUrl: "/images/kitten.png",
+    bio: "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+    githubUrl: "https://github.com/AnnaPestova1"
+  },
+  {
+    name: "Anna Solovykh",
+    role: "Front-end",
+    imageUrl: "/images/AnnaSolovykh.jpeg",
+    bio: "A web developer specializing in Frontend (JS, React, Redux), wishing to improve and master.",
+    githubUrl: "https://github.com/AnnaSolovykh"
+  },
+  {
+    name: "Aigul Yedigeyeva",
+    role: "Back-end",
+    imageUrl: "/images/kitten.png",
+    bio: "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+    githubUrl: "https://github.com/AigulYe"
```

Comment:
Could you please fix the link. This link is not work. I suppose the correct one is: `https://github.com/AigulY`